src/io/windows.zig

```diff
@@ -1394,8 +1394,19 @@ pub const IO = struct {
             };
         }
 
+        // Declaring the function with an alternative name because `CamelCase` functions are
+        // by convention, used for building generic types.
+        const set_end_of_file = @extern(
+            *const fn (
+                hFile: os.windows.HANDLE,
+            ) callconv(os.windows.WINAPI) std.os.windows.BOOL,
+            .{
+                .library_name = "kernel32",
+                .name = "SetEndOfFile",
+            },
+        );
```

Comment:
I am curious here, what's the context of these being removed from std? As in, should we perhaps be using something else?

Reply:
This:
https://github.com/ziglang/zig/commit/3095e83d1162081ba54ed73c59810bfe0fb3f025

Reply:
> should we perhaps be using something else?

I've added a `TODO(zig)` for this one (planning to take a look after the migration)
https://github.com/tigerbeetle/tigerbeetle/pull/2814/commits/f240d7dcdc3622b09de6bdbd87f914b8b876d83b